src/utils.ts

```diff
@@ -128,8 +128,12 @@ const getSentryErrorReporter = (dsn: string): ((m: string) => void) => {
   }
 }
 
+const HEX_PREFIX = '0x'
 const isDefined = (arg: any) => arg !== undefined && arg !== null
+const prepend0x = (i: string) =>
+  !i.startsWith(HEX_PREFIX) ? `${HEX_PREFIX}${i}` : i
```

Comment:
- Can reverse the branches to avoid the `!` and keep consistency with `remove0x`.
- If we are naming `0x` as the "hex prefix", we can also carry the name to `prepend` and `remove`.

```suggestion
const prependHexPrefix = (i: string) =>
  i.startsWith(HEX_PREFIX) ?  i : `${HEX_PREFIX}${i}`
```

Reply:
incorporated suggestions, please check.

Reply:
looks great 👍🏼

Reply:
Actually since we are using hexPrefix, changed functions to `addHexPrefix` and `removeHexPrefix`, makes more sense now.